lib/java/test-utils/src/main/java/org/enso/test/utils/TestUtils.java

```diff
@@ -36,24 +34,28 @@
 import org.graalvm.polyglot.Value;
 import org.graalvm.polyglot.io.IOAccess;
 import org.graalvm.polyglot.proxy.ProxyExecutable;
-import org.junit.rules.TemporaryFolder;
 
-public abstract class TestBase {
-  protected static Context createDefaultContext() {
+/**
+ * A collection of classes and methods useful for testing, mostly Truffle-related stuff.
+ */
+public final class TestUtils {
```

Comment:
While it is good to split things into modules, the `TestUtils` object feels like an antipattern in this case. It violates the object-oriented design - one object for one thing. And it can quickly become bloated by accumulating more random methods.

Instead of creating one bag-of-things, I would split `TestUtils` into separate objects based on their functionality. From what I can see, 
1. there are a bunch of methods that work with a context. I would create a `TestContexts` or `ContextUtils` if you like this kind of naming
2. there are some methods that work with the project. They can go into a `TestProjects` or `ProjectUtils`

Reply:
Good idea. Done in c9ded92d9a6f9a52e6a7f6c3018f79d53baff035

Reply:
`ContextUtils` is better name than `TestUtils`.